lib/prompts/sandboxesPrompt.ts

```diff
@@ -56,17 +62,19 @@ const sandboxTypePrompt = () => {
       default: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
     },
   ]);
-};
+}
 
-const deleteSandboxPrompt = (promptParentAccount = false) => {
+export function deleteSandboxPrompt(
+  promptParentAccount = false
+): Promise<GenericPromptResponse | undefined> {
```

Comment:
Why are you using `GenericPromptResponse` here rather than making one?

Reply:
Didn't check GitHub Copilot enough 🤦‍♀️

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/1d3fbba59163fc76535c881196f18e41623b8c4b